### PR TITLE
[bug 1121588] Update django-cache-machine for django 1.7.

### PR DESCRIPTION
**Pull request by rlr**
This has some fixes for the new django. We can probably ditch cache machine for real since we dont really use it for what it is meant.

r?

### CODE REVIEW

File: requirements/default.txt [hunk 1]
```diff
@@ -67,8 +67,8 @@ https://github.com/django-security/django-axes/archive/778f208cc1d3cc42ecb756f73
 https://github.com/mozilla/django-badger/archive/3e9fded1a5bd17a69a852d410eaa4b79686bdd32.tar.gz#egg=django-badger==0.0.0.1
 
 # django-cache-machine: master~1
```
Comment (willkg) [inline requirements/default.txt]: Is this still "master tip minus one commit"? If not, then I'd nix the `~1` part on this line.

**Comment by rlr** [comment]
good point. fixord :point_up:

**Comment by rlr** [comment]
The commits:
https://github.com/django-cache-machine/django-cache-machine/compare/7c3d3abfc85fc26d0e522fd05e9a42fc9b9a5430...6b2b37ebbcac85a9f86e0e3b64c1c54243f50006

It's django 1.7 compatibility and other small fixes from what I can tell.

**Comment by mythmon** [comment]
Looks good. Especially because we aren't actually using it much.

**Pull request merged**